models/ssh_key.go

```diff
@@ -104,8 +105,10 @@ func extractTypeFromBase64Key(key string) (string, error) {
 
 // parseKeyString parses any key string in OpenSSH or SSH2 format to clean OpenSSH string (RFC4253).
 func parseKeyString(content string) (string, error) {
-	// Transform all legal line endings to a single "\n".
-	content = strings.NewReplacer("\r\n", "\n", "\r", "\n").Replace(content)
+	// Transform all legal line endings to a single "\n"
+	newlineExpression := regexp.MustCompile(`\r?\n`) // windows style optional with (\r?)
+	content = newlineExpression.ReplaceAllString(content, "\n")
```

Comment:
well, `\r?\n` will only match `\r\n` and `\n`, not `\r` like the previous implementation... I think you're looking for `\r\n?` in that case